Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/WANDPowderReductionTest.py

```diff
@@ -334,6 +340,13 @@ def test_event(self):
             BankPixelWidth=100,
             WorkspaceType="Event",
         )
+        event_data2 = CreateSampleWorkspace(
```

Comment:
Creating a second input workspace will have different reduction results than the first. Just supply it twice like the other tests.